frontend/demo/component/badge/badge-icons-only-table.ts

```diff
@@ -1,22 +1,88 @@
+import { getUserPermissions } from 'Frontend/demo/domain/DataService'; // hidden-full-source-line
 import 'Frontend/demo/init'; // hidden-full-source-line
+import UserPermissions from 'Frontend/generated/com/vaadin/demo/domain/UserPermissions'; // hidden-full-source-line
 
-import { html, LitElement, customElement } from 'lit-element';
-import '@vaadin/vaadin-checkbox/vaadin-checkbox';
+import '@vaadin/vaadin-grid';
+import '@vaadin/vaadin-icons/vaadin-icons';
+import '@vaadin/vaadin-lumo-styles/icons';
```

Comment:
Unnecessary import of Lumo icons as only Vaadin icons are used here.
```suggestion
import '@vaadin/vaadin-icons/vaadin-icons';
```